designer/server/src/models/forms/library.js

```diff
@@ -78,9 +83,19 @@ function buildPaginationPages(currentPage, totalPages, perPage) {
    * @returns {PaginationPage} The pagination page item.
    */
   function createPageItem(pageNumber, isCurrent = false) {
+    const queryParams = new URLSearchParams({
+      page: pageNumber.toString(),
+      perPage: perPage.toString()
+    })
+
+    if (sorting?.sortBy && sorting.order) {
+      const field = sorting.sortBy === 'updatedAt' ? 'updated' : 'title'
+      queryParams.set('sort', `${field}_${sorting.order}`)
+    }
+
     return {
       number: String(pageNumber),
-      href: `${formsLibraryPath}?page=${pageNumber}&perPage=${perPage}`,
+      href: `${formsLibraryPath}?${queryParams.toString()}`,
```

Comment:
Was ESLint kicking off without the `.toString()`?

```suggestion
      href: `${formsLibraryPath}?${queryParams}`,
```

Reply:
Ah that's a remnant I forgot to remove. We're covered by line-86-89:
```
    const queryParams = new URLSearchParams({
      page: pageNumber.toString(),
      perPage: perPage.toString()
    })
```